src/Microsoft.VisualStudio.Editors/PropPages/AssemblyInfoPropPage.resx

```diff
@@ -1122,21 +1038,9 @@
   <data name="FileVersionRevisionTextBox.TabIndex" type="System.Int32, mscorlib">
     <value>3</value>
   </data>
-  <data name="&gt;&gt;FileVersionRevisionTextBox.Name" xml:space="preserve">
-    <value>FileVersionRevisionTextBox</value>
-  </data>
-  <data name="&gt;&gt;FileVersionRevisionTextBox.Type" xml:space="preserve">
-    <value>System.Windows.Forms.TextBox, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
-  </data>
-  <data name="&gt;&gt;FileVersionRevisionTextBox.Parent" xml:space="preserve">
-    <value>FileVersionLayoutPanel</value>
-  </data>
-  <data name="&gt;&gt;FileVersionRevisionTextBox.ZOrder" xml:space="preserve">
-    <value>3</value>
-  </data>
-  <metadata name="$this.Localizable" type="System.Boolean, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089">
+  <data name="$this.Localizable" type="System.Boolean, mscorlib">
```

Comment:
Do you know what's going on with this `<metadata>` -> `<data>` change?

Reply:
I do not, who could I ask about this? It happen whenever a .resx is changed.

Reply:
I do not, who could I ask about this? It happens whenever a .resx is changed.